data/repository/src/commonMain/kotlin/social/androiddev/common/repository/timeline/HomeTimelineRepository.kt

```diff
@@ -0,0 +1,37 @@
+package social.androiddev.common.repository.timeline
+
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.distinctUntilChanged
+import org.mobilenativefoundation.store.store5.Market
+import org.mobilenativefoundation.store.store5.MarketResponse
+import org.mobilenativefoundation.store.store5.ReadRequest
+import social.androiddev.common.timeline.TimelineItem
+
+interface HomeTimelineRepository {
+    suspend fun read(): Flow<MarketResponse<List<TimelineItem>>>
+}
+
+class RealHomeTimelineRepository(
```

Comment:
I think I like the "Real*" nomenclature over "*Impl". We should be consistent on this though throughout the project

Reply:
Yeah I wanted to bring up, I followed square libraries that did this years ago and kinda stuck with it. Let's discuss in slack and if folks are fine I can refactor everything we have. 

Real is nice because you can have:
RealFoo
FakeFoo
UserFoo 
etc

Reply:
yup, that's the pattern I am seeing right now and also growing very fond of it 😄